docs-app/ember-cli-build.js

```diff
@@ -14,10 +18,50 @@ module.exports = function (defaults) {
     },
   });
 
-  return require('@embroider/compat').compatBuild(app, Webpack, {
+  const options = {
     packagerOptions: {
+      cssLoaderOptions: {
+        modules: {
+          localIdentName: isProduction()
+            ? '[sha512:hash:base64:5]'
+            : '[path][name]__[local]',
+          // Enable local mode only for CSS files from the host app
+          mode: (resourcePath) => {
+            // The host app and active child addons are moved into a common
+            // stable temp dir (`options.workspaceDir`), before the `css-loader`
+            // processes them.
+            //
+            // We want to enable local mode only for our own host app. All other
+            // addons should be loaded in global mode.
+            const hostAppWorkspaceDir = `${options.workspaceDir}/${app.name}`;
+            const isHostAppPath = resourcePath.includes(hostAppWorkspaceDir);
+
+            return isHostAppPath ? 'local' : 'global';
+          },
+        },
+        sourceMap: !isProduction(),
+      },
+      publicAssetURL: '/',
       webpackConfig: {
-        // ...
+        module: {
+          rules: [
+            {
+              exclude: /node_modules/,
+              test: /\.css$/i,
+              use: [
+                {
+                  loader: 'postcss-loader',
+                  options: {
+                    sourceMap: !isProduction(),
+                    postcssOptions: {
+                      config: './postcss.config.js',
+                    },
+                  },
+                },
+              ],
+            },
+          ],
+        },
```

Comment:
For more options, including how to configure `url-loader`, see https://github.com/evoactivity/ember-modern-css#webpack-configuration.